cv.tex

```diff
@@ -29,9 +33,11 @@
 
 \lastupdated % Print the Last Updated text at the top right
 
-\namesection{Debarghya}{Das}{ % Your name
-\urlstyle{same}\url{http://debarghyadas.com} \\ % Your website, LinkedIn profile or other web address
-\href{mailto:dd367@cornell.edu}{dd367@cornell.edu} | 607.379.5733 % Your contact information
+\namesection{Joe}{Pate}{ % Your name
```

Comment:
(JC):
Should you include a location (i.e., Seattle, WA)?

Reply:
done